src/synthesis/autocorrelation.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+from src.conf import config
+import operator
+import math
+
+def acorr(window, fs = config.analysis['sampling_frequency'], fmin = config.analysis['fmin'], fmax = config.analysis['fmax']):
```

Comment:
kurde, sprytnie to zrobiłaś, nie wpadłem na takie dawanie configowych parametrów wcześniej xd